docs/docs/atom-archive/getting-started/sections/installing-atom.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Installing Atom
+---
+### Installing Atom
+
+To get started with Atom, we'll need to get it on your system. This section will go over installing Atom on your system as well as the basics of how to build it from source.
+
+Installing Atom should be fairly simple. Generally, you can go to https://atom.io and you should see a download button as shown here:
+
+{{#mac}}
+
+![Download buttons on https://atom.io](../../images/mac-downloads.png "Download buttons on https://atom.io")
+
+{{/mac}}
+
+{{#windows}}
+
+![Download buttons on https://atom.io](../../images/windows-downloads.png "Download buttons on https://atom.io")
+
+{{/windows}}
+
+{{#linux}}
+
+![Download buttons on https://atom.io](../../images/linux-downloads.png "Download buttons on https://atom.io")
```

Comment:
Same as above, regarding `title` and `alt` attributes. In this case the URL provides no real benefit (do you want people to type it into their address bar? Why not use ”…on the Atom homepage” instead?)